### PR TITLE
Adds 1/3 sec delay to putting people in cryo

### PR DESCRIPTION
**Pull request by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE**
1 sec if lying, 3 secs if standing

It's a bit crazy that you can instasleep people if they're standing next to cryo, like an infinitely reusable trickypen, this gives them a small amount of time to react at least
Delay is almost nothing for people that are already stunned/in crit

### CODE REVIEW

**Comment by DamianX** [comment]
I died

**Comment by ShiftyRail** [comment]
Muh efficiency

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
@DamianX actually I killed somebody :^)

**Comment by aceedex** [comment]
I am fairly certain you can right-click eject and walk it off in the few seconds it tales to sleep you

**Comment by Blithering** [comment]
If you're holding a movement key, you will just walk straight out of the cryo again when shoved inside. I'd prefer a delay on the sleep to delaying putting people inside.

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
@aceedex It sleeps you in one tick, meaning a random amount between 0 and 2 seconds

**Comment by ShiftyRail** [comment]
@aceedex Or even alt-click then there is a shortcut.

**Comment by Blithering** [comment]
Gonna clarify, I think its the preferable option as it doesn't mean doctors have to stand there waiting for the bar to fill and can treat other patients or go grab an IV, also means in the event the patient is already asleep due to crit or something it can start the effect immediately, minimises the change to normal cryo procedure while still making combat tubing less of a thing

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
I'd have to change how cryo coldness works because it's the cold that makes you sleep, and so it wouldn't start faster for people in crit because they would have to be cooled down too. Kind of tricky

Anyways, it's just 1 second for lying people which is 80% of the people you want to cryo

**Comment by Chazach** [comment]
fuck off,  tubing is a feature

**Comment by Irradiation** [comment]
@I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE Is there a progress bar? I want a motherfucking progress bar if this gets merged. Also GOTTA GO FAST, NO TIME TO WASTE. 

As for the infinite trickypenning thingy, you've already got a non-negligible chance to dash out of the cryo tube (Happened plenty of times to me) if somebody dunked you into it even if you haven't seen it coming, thus providing some tangible counterplay (unlike trickypens).

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
There's TWO progress bars!

**Comment by Irradiation** [comment]
Gee coder, how come the collabs let you have TWO progress bars?

**Comment by boy2mantwicethefam** [comment]
@I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE NOOOOOOOOO MUH VALIDS HOW AM I SUPPOSED TO DEFEND MYSELF AS A DOCTOR NOW??? THIS CANT BE HAPPENING NOW YOU CANT USE EVEN MORE STUNS??? HOW AM I SUPPOSED TO STRIP AND USE RESTRAINTS AND AMPUTATE EVERY LIMB LIVING OUT MY SICK DESIRES NOW????????????
+1

**Comment by aceedex** [comment]
Personally don't touch it because its hilarious to me to dunk a wiznigga in cryo and they don't try to move

**Comment by Irradiation** [comment]
Extra hilarious when you dunk a xenomorph into the magic healing tube.

File: code/game/machinery/cryo.dm [hunk 1]
```diff
@@ -563,39 +558,46 @@ var/global/list/cryo_health_indicator = list(	"full" = image("icon" = 'icons/obj
...
-		M.drop_item(I) // to avoid visual fuckery bobing. Doesn't do anything to items with cant_drop to avoid magic healing tube abuse.
-	//M.metabslow = 1
-	add_fingerprint(usr)
-	update_icon()
-	nanomanager.update_uis(src)
-	M.ExtinguishMob()
-	return 1
+
+	if(M != usr)
+		usr.visible_message("<span class='warning'>[usr] starts putting [M.name] into \the [src].</span>", "<span class='notice'>You start putting [M.name] into the [src].</span>", "You hear some glass knocking.")
```
Comment (DamianX) [inline code/game/machinery/cryo.dm:575]: `the [src]` -> `\the [src]`

**Comment by strawberrytan** [comment]
rip anything in medbay that can be used to kill someone

it's already something you can escape from

**Comment by ShiftyRail** [comment]
Tubing requires a sufficient amount of vivacity and alertness to pull off, it should be rewarded
All I see to this is a way to make the wild ride clunkier with an unneeded bottleneck near the tube

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
I considered this to be a very minor change either way if it passed or not, so the attention it's getting is probably better spent on other PRs, I don't mind

**Pull request closed without merging**